decidim-core/app/views/decidim/shared/_tags.html.erb

```diff
@@ -1,6 +1,9 @@
 <% if resource.category.present? || has_visible_scopes?(resource) %>
   <ul class="tags <%= tags_class_extra %>" >
     <% if resource.category.present? %>
+      <% if resource.previous_category.present? && action_name == "show" %>
+        <span style="text-decoration: line-through"><%= resource.previous_category.translated_name %></span>
```

Comment:
Is this design agreed with product? 😕 I mean, From the screenshots I didn't realize this was a category change, to me it looks like a weirdly formatted line from the body 😕

Reply:
Design is pending, I've opened #2807.
I'll remove it.